package.json

```diff
@@ -3,7 +3,7 @@
     "node": "20.*.*"
   },
   "name": "guac-visualizer",
-  "version": "main",
+  "version": "0.0.0-development",
```

Comment:
Out of curiosity, why the change from `main` to `0.0.0-development`? The version insertion in the release flow handles `main`. Is this a style preference (which is totally fine) or is there a somewhere else that needs a more version-looking format?

Reply:
Indeed, rather a style preference, and three 'light' arguments:
* `0.0.0-development` is easier to (full-text-)search in the codebase
* `main` assumes higher knowledge that a specific git branch is meant. Also, it could be that a version was build from another branch, which then would be misleading.
* the proposed string is hopefully in a more version-ish format so more people realize it as such.

Reply:
Works for me!